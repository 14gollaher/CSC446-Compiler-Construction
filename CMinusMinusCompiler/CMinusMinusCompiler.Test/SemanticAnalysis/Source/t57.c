int a,b;
int a(int x, int y) 
{
	return -1;
}
int main()
{
  int a;
  return -1;
}
