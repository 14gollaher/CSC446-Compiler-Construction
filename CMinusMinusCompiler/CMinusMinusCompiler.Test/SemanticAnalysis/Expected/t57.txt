ERROR: Line 6 Duplicate lexeme "a" with depth "1" exists
Completed processing t57.c
